configtx/orderer.go

```diff
@@ -185,6 +195,288 @@ func (o *OrdererGroup) Configuration() (Orderer, error) {
 	}, nil
 }
 
+// BatchSize returns orderer configuration's batch size values.
```

Comment:
`// BatchSize returns a BatchSizeValue that can be used to configure an orderer configuration's batch size parameters.`